app/components/base_field/base_field.html.erb

```diff
@@ -1,6 +1,10 @@
 <div <%= attrs %>>
   <div class="BaseField-textWrapper <%= hide_label ? 'visually-hidden' : '' %>">
-    <label class="BaseField-labelText" for="<%= id %>">
+    <label
+      class="BaseField-labelText"
+      for="<%= id %>"
+      id="label-<%= id %>"
```

Comment:
Why is this necessary?

Reply:
Clippy forever

Reply:
> Why is this necessary?

It allows us to reference the label in the `aria-describedby` attribute. This ensures that screenreaders announce something like "Group: Header-Bild" when you focus the image input.